app/models/invoice.rb

```diff
@@ -5,4 +5,24 @@ class Invoice < ApplicationRecord
   has_many :transactions
 
   validates :status, presence: true
+
+  def self.get_by_key(key, params)
```

Comment:
@s-espinosa My understanding is that you can put methods in your ApplicationRecord that can be shared across models. If this is the case, would you recommend putting these methods there?